plugins/mcreator-localization/lang/texts.properties

```diff
@@ -184,6 +184,12 @@ dialog.gui.model_rotation_x=Model yaw rotation in degrees
 dialog.gui.model_follow_mouse=Follow mouse movement with rotation
 dialog.gui.add_tooltip=Tooltip editor
 dialog.gui.tooltip_display_condition=Tooltip display condition
+dialog.gui.label_anchor=Anchor to:
```

Comment:
Another element is to make those texts general texts for all components, so we don't duplicate texts saying the same thing.

An example of they key can be: dialog.gui.anchor

Reply:
LGTM